src/main/java/mpern/sap/commerce/build/HybrisPluginExtension.java

```diff
@@ -29,6 +35,8 @@ public HybrisPluginExtension(Project project) {
         antTaskDependencies = project.getObjects().listProperty(Object.class);
 
         platform = project.getObjects().newInstance(HybrisPlatform.class, project);
+
+        sparseBootstrap = project.getObjects().newInstance(SparseBootstrap.class);
```

Comment:
There are nicer ways to inject this as a nested configuration, but they do not work with gradle 5.x.